File: src/org/irislang/jiris/core/exceptions/fatal/IrisTypeNotCorretException.java
package org.irislang.jiris.core.exceptions.fatal;

/**
 * Created by Huisama on 2017/5/26 0026.
 */
public class IrisTypeNotCorretException extends IrisFatalException {
    public IrisTypeNotCorretException(String fileName, int lineNumber, String message) {
        super(fileName, lineNumber, message);
    }

    @Override
    public String GetFatalExceptionName() {
        return "TypeNotCorrectIrregular";
    }
}

File: src/main/java/org/irislang/jiris/core/exceptions/fatal/IrisTypeNotCorretException.kt
package org.irislang.jiris.core.exceptions.fatal

/**
 * Created by Huisama on 2017/5/26 0026.
 */
class IrisTypeNotCorretException(fileName: String, lineNumber: Int, message: String) : IrisFatalException(fileName, lineNumber, message) {

    override fun GetFatalExceptionName(): String {
        return "TypeNotCorrectIrregular"
    }
}
